lib/index.js

```diff
@@ -74,10 +76,20 @@ export default function rehypePicture(options) {
 
       for (key in map) {
         if (Object.hasOwn(map, key)) {
+          /** @type {{srcSet: string, type: string | null | undefined, width?: string, height?: string}} */
+          const prop = {srcSet: replaceExt(src, '.' + key), type: map[key]}
+          if (width) {
+            prop.width = String(width)
+          }
+
+          if (height) {
+            prop.height = String(height)
+          }
+
           sources.push({
             type: 'element',
             tagName: 'source',
-            properties: {srcSet: replaceExt(src, '.' + key), type: map[key]},
+            properties: prop,
```

Comment:
```suggestion
            properties,
```

Then you can shorten this here